source/common/access_log/access_log_impl.cc

```diff
@@ -54,7 +54,7 @@ bool ComparisonFilter::compareAgainstValue(uint64_t lhs) const {
 }
 
 FilterPtr FilterFactory::fromProto(const envoy::config::accesslog::v3::AccessLogFilter& config,
-                                   Server::Configuration::FactoryContext& context) {
+                                   Server::Configuration::GenericFactoryContext& context) {
```

Comment:
+1: will need listenerInfo for our-use case.

Reply:
Before https://github.com/envoyproxy/envoy/pull/31012, how your code handle this case. At that time, we only have `CommonFactoryContext` for log filters.

(But I have no opinion to this, it completely up to you. :)